spec/swagger.yaml

```diff
@@ -4783,5 +5028,548 @@ definitions:
+                    example: '/pub/health_profile/59d441ac21741161bbe266b5/questions'
+    example:
+      {
+        "type": "health_profile",
+        "id": "59d441ac21741161bbe266b5",
+        "attributes": {
+          "created_at": "2017-07-07T19:15:56.229Z",
+          "stats": [
+            {"category": "lifestyle_behaviors", "answered": 5, "total": 11},
+            {"category": "mental_wellbeing", "answered": 3, "total": 5},
+            {"category": "preventative_care", "answered": 1, "total": 3},
+            {"category": "overall", "answered": 9, "total": 19}
+          ],
+          "updated_at": "2017-07-07T19:15:56.229Z",
+        },
+        "relationships": {
+          "patient": {
+            "data": {
+              "id": "595fddea28c5041b84466bbe",
+              "type": "patient"
+            },
+            "links": {
+              "related": "/pub/health_profile/59d441ac21741161bbe266b5/patient"
+            }
+          },
+          "questions": {
+            "data": [
+              {
+                "id": "595fddea28c5041b84466bbe",
+                "type": "health_profile_question"
+              }
+            ],
+            "links": {
+              "related": "/pub/health_profile/59d441ac21741161bbe266b5/questions"
+            }
+          }
+        },
+        "links": {
+            "self": "/pub/health_profile/59d441ac21741161bbe266b5"
+        },
+      }
+
+  FetchHealthProfileResponse:
+    type: object
+    required:
+      - data
+    properties:
+      meta:
+        $ref: '#/definitions/FetchMetaResponse'
+      data:
+        $ref: '#/definitions/HealthProfileResource'
+      included:
+        description: |
+          Related resources which are included in the response based on the `include` param.
+          Attributes of each resource will vary depending on the type.
+          See [action](#operation/fetchHealthProfileQuestion), [bundle](#operation/fetchHealthProfileAnswer)
+        type: array
+        items:
+          $ref: '#/definitions/Resource'
+
+  FetchHealthProfilesResponse:
+    type: object
+    required:
+      - data
+    properties:
+      meta:
+        $ref: '#/definitions/FetchMetaResponse'
+      links:
+        $ref: '#/definitions/CollectionResponseLinks'
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/HealthProfileResource'
+      included:
+        description: |
+          Related resources which are included in the response based on the `include` param.
+          Attributes of each resource will vary depending on the type.
+          See [action](#operation/fetchHealthProfileQuestion), [bundle](#operation/fetchHealthProfileAnswer)
+        type: array
+        items:
+          $ref: '#/definitions/Resource'
+
+  HealthProfileQuestionResource:
+    type: object
+    required:
+      - type
+      - id
+    properties:
+      type:
+        type: string
+      id:
+        type: string
+      attributes:
+        type: object
```

Comment:
The absence of attributes probably warrants some explanation. Or perhaps attributes should be omitted? The example does not include `data.attributes`.

Reply:
Resolved.